packages/universal-components/src/TagsInput/components/Tag.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { Text } from '../../Text';
+import { StyleSheet, type StylePropType } from '../../PlatformStyleSheet';
+import { designTokens } from '../../DesignTokens';
+import { Icon } from '../../Icon';
+import { ExtendedTouchable } from '../../ExtendedTouchable';
+
+type Props = {|
+  +children: React.Node,
+  +style?: StylePropType,
+  +testID?: string,
+  +index: number,
+  +onDeletePress?: number => void,
+  +fontSize: number,
+|};
+
+type TouchableTagIconProps = {|
+  +onPress?: number => void,
+  +children: React.Node,
+  +index: number,
+|};
+
+class TouchableTagIcon extends React.Component<TouchableTagIconProps> {
```

Comment:
What about making in functional component?

Reply:
(eslint is complaining, we agreed that we will keep class component for now)